mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -0,0 +1,320 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cast"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/mwielbut/pointy"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorSnapshotBackupScheduleUpdate  = "error updating a Cloud Backup Schedule: %s"
+	errorSnapshotBackupScheduleRead    = "error getting a Cloud Backup Schedule for the cluster(%s): %s"
+	errorSnapshotBackupScheduleSetting = "error setting `%s` for Cloud Backup Schedule(%s): %s"
+)
+
+// https://docs.atlas.mongodb.com/reference/api/cloud-backup/schedule/modify-one-schedule/
+// same as resourceMongoDBAtlasCloudProviderSnapshotBackupPolicy
+func resourceMongoDBAtlasCloudBackupSchedule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMongoDBAtlasCloudBackupScheduleCreate,
+		Read:   resourceMongoDBAtlasCloudBackupScheduleRead,
+		Update: resourceMongoDBAtlasCloudBackupScheduleUpdate,
+		Delete: resourceMongoDBAtlasCloudBackupScheduleDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceMongoDBAtlasCloudBackupScheduleImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"policies": {
+				Type:     schema.TypeList,
```

Comment:
Understood  thank you, I changed to TypeSet with Custom hashing because of ID, added Max/Min Items for policies and items

Reply:
Ah, I missed this response in the review since it wasn't in context in the code.   That wasn't quite what I was trying to express with the 0-4 items but may have captured it better in my current comments.   Let me know.